website/docs/cli/commands/apply.mdx

```diff
@@ -7,105 +7,86 @@ description: >-
 
 # Command: apply
 
-> **Hands-on:** Try the [Terraform: Get Started](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn. For more in-depth details on the `apply` command, check out the [Apply Terraform Configuration tutorial](https://learn.hashicorp.com/tutorials/terraform/apply?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS).
-
 The `terraform apply` command executes the actions proposed in a Terraform
 plan.
 
-The most straightforward way to use `terraform apply` is to run it without
-any arguments at all, in which case it will automatically create a new
-execution plan (as if you had run `terraform plan`) and then prompt you to
-approve that plan, before taking the indicated actions.
-
-Another way to use `terraform apply` is to pass it the filename of a saved
-plan file you created earlier with `terraform plan -out=...`, in which case
-Terraform will apply the changes in the plan without any confirmation prompt.
-This two-step workflow is primarily intended for when
-[running Terraform in automation](https://learn.hashicorp.com/tutorials/terraform/automate-terraform?in=terraform/automation&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS).
+> **Hands On:** Try the [Apply Terraform Configuration](https://learn.hashicorp.com/tutorials/terraform/apply?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial to for details about how Terraform applies a configuration, how Terraform recovers from errors during apply, and common ways to use this command.
 
 ## Usage
 
 Usage: `terraform apply [options] [plan file]`
 
-The behavior of `terraform apply` differs significantly depending on whether
-you pass it the filename of a previously-saved plan file.
+
 
 ### Automatic Plan Mode
 
-In the default case, with no saved plan file, `terraform apply` creates its own
-plan of action, in the same way that [`terraform plan`](/cli/commands/plan) would.
+When you run `terraform apply` without arguments, Terraform automatically creates a new execution plan as if you had run [`terraform plan`](/cli/commands/plan), prompts you to approve that plan, and takes the indicated actions. You can use all of the [planning modes](/cli/commands/plan#planning-modes) and
```

Comment:
Tricky here — it also runs in automatic plan mode if you pass it any number of options, like the plan modes/options we immediately mention. From a CLI taxonomy perspective you could argue flags/options aren't "arguments", but it might be best to avoid the ambiguity. 

```suggestion
When you run `terraform apply` without passing a saved plan file, Terraform automatically creates a new execution plan as if you had run [`terraform plan`](/cli/commands/plan), prompts you to approve that plan, and takes the indicated actions. You can use all of the [planning modes](/cli/commands/plan#planning-modes) and
```